pkg/asset/installconfig/vsphere/vsphere.go

```diff
@@ -94,35 +97,43 @@ func Platform() (*vsphere.Platform, error) {
 // If creating the client fails, an error is returned.
 func getClients() (*vCenterClient, error) {
 	var vcenter, username, password string
-	survey.Ask([]*survey.Question{
+	var err error
+
+	if err = survey.Ask([]*survey.Question{
```

Comment:
no need to use an existing err, it just shadows the err and can cause over writes when not desired.

```suggestion
	var err error

	if err := survey.Ask([]*survey.Question{
```

Reply:
oh right! updated